### DISCUSSION TITLE
How make metadata url(text) as hyperlink.

### DISCUSSION THREAD

**Discussion by jeshani** [doccano/doccano#1483] [Q&A]
![image](https://user-images.githubusercontent.com/56191568/131225933-abeca811-481a-4378-8343-5f99d809b94f.png)

I want to invoke the reference pdf directly from metadata, Is there any way to make this URL text as hyperlink?

Uploaded JSONL ex. 
{"data": "Here is the texts extracted from 35196.pdf", "label": [], "Id": "35196.pdf", "url": "file:///D:/Documents/35196.pdf"}

**Comment by PedroMTQ** [comment]
I have the same question, did you manage to figure it out?

**Comment by david-engelmann** [comment]
If you use the doccano-client each sample is associated with its upload document. As far as making the internal upload document as downloadable from the Metadata section, that will be difficult. Might be easiest to add the link to a external hosted version of the document, such as a Dropbox url or gsheet link
